### PR TITLE
ONL-5095 add contributing file and update readme file

### PR DESCRIPTION
**Pull request by jaimeperezpedra**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
+# Contributing
+
+For the development of @ebury/chameleon-components, we use storybook [Storybook](https://storybook.js.org/).
+If you want to contribute to the library then you must do the following to set up your local environment:
+
+## Storybook
+
+1\. Clone chameleon-components repo
```
Comment (matallo) [inline CONTRIBUTING.md:8]: this is the instalation guide and goes in the readme https://bitbucket.org/fxsolutions/eburyonline/src/master/README.md

**Review by matallo — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+* The PR is considered **approved** when **some code owner** have approved the PR.
+* The comments can be done at PR level or at detailed level.
+* The comments can be in different levels of relevance: **mandatory** or **optional**.
+    * The mandatory changes must have a associated **task** in the Pull Request view.
+    * All the mandatory changes must be changed and the associated tasks marked as **done**.
+* The code owners review the manual checks for **Style guide** of the project.
+* If the change is rejected by the code owners and there is not possibility of modifications, the PR will be marked as **declined**, and the process do not continue. If it can be modified, the Pull Request should not be declined, only commented with mandatory changes.
+* The code review phase is considered **finished** when all the mandatory changes are done and it is approved.
+
+### Merge conditions
```
Comment (jmesao) [inline CONTRIBUTING.md:205]: @jaimeperezpedra Should we mention if we should follow squash strategy when we are merging?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jaimeperezpedra) [reply CONTRIBUTING.md:8]: Here there are two things, the installation to develop/contribute on the library or the installation to use the library in other projects, I guess the one to develop should be on the contributing and the other one on the README, and we already have these two installation process on the files.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jaimeperezpedra) [reply CONTRIBUTING.md:205]: done!

File: README.md [hunk 3]
```diff
@@ -14,12 +14,33 @@ Also, check out the latest version deployed in [Storybook](https://chameleon.ebu
...
 
+### Install the library in your project
+
 Install @ebury/chameleon-components in your project with npm:
 
 ```sh
 npm install @ebury/chameleon-components --save
 ```
 
+### Storybook
```
Comment (matallo) [inline README.md:25]: I don't think this is the best title

File: README.md [hunk 4]
```diff
@@ -14,12 +14,33 @@ Also, check out the latest version deployed in [Storybook](https://chameleon.ebu
 
 ## Installation
 
+### Install the library in your project
```
Comment (matallo) [inline README.md:17]: Probably wrong order, first I'd have project installation instruction, then I'd have this as part of the usage section? Ask for feedback in Slack

**Review by matallo — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (jaimeperezpedra) [reply README.md]: I changed it

File: README.md [hunk 5]
```diff
@@ -49,6 +39,15 @@ See the [contributing](CONTRIBUTING.md) docs for more info on code style, format
...
 ## Usage
 
+### Install the library in your project
+
+Install @ebury/chameleon-components in your project with npm:
+
+```sh
+npm install @ebury/chameleon-components --save
+```
+### Import component/s
```
Comment (matallo) [inline README.md:49]: Empty line?

**Review by matallo — APPROVED** [review]
Minor edit and approved!

**Review by matallo — APPROVED** [review]

**Review by stavros-tomas — APPROVED** [review]

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,7 +1,38 @@
...
+1. [Submitting changes](#markdown-header-submitting-changes)
+    1. [Creating a PR](#markdown-header-creating-a-pr)
+    1. [Code Review](#markdown-header-code-review)
+    1. [Merge conditions](#markdown-header-merge-conditions)
+1. [Release](#markdown-header-release)
+1. [Reporting a problem or requesting a change](#markdown-header-reporting-a-problem-or-requesting-a-change)
+1. [How to report a bug](#markdown-header-how-to-report-a-bug)
+1. [Thanks](#markdown-header-thanks)
+
+Hi there! We’d love your help if you're interested in contributing to Eburyonline. Eburyonline is developed by the [Online team](https://fxsolutions.atlassian.net/wiki/spaces/TEAM/pages/119543461/ONL) members. Check the [VISION](VISION.md) for the project as well as the guidelines below to contribute.
```
Comment (matallo) [inline CONTRIBUTING.md:23]:
```suggestion
Hi there! We’d love your help if you're interested in contributing to Chameleon. Chameleon is the design system developed by the [Online team](https://fxsolutions.atlassian.net/wiki/spaces/TEAM/pages/119543461/ONL) members for Eburyonline. Check the [VISION](VISION.md) for the project as well as the guidelines below to contribute.
```

**Review by matallo — APPROVED** [review]

**Review by matallo — APPROVED** [review]

**Pull request merged**